maintenance/removePII.php

```diff
@@ -636,23 +636,23 @@ public function execute() {
 		);
 
 		$user = User::newSystemUser( 'MediaWiki default', [ 'steal' => true ] );
+
+		$UserGroupManager = MediaWikiServices::getInstance()->getUserGroupManager();
```

Comment:
Should use coding convention e.g. camelCasing.

Reply:
oh, right, thanks!

Reply:
fixed

Reply:
doesn't seem fixed to me, still showing upper case.

Reply:
ok, now it's fixed, sorry